frontend/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,26 @@
+import { makeCrumb } from './makeCrumb';
+import { BreadCrumbProps, BreadCrumbStyles, CrumbProperty } from './types';
+import { styleMapper } from './styleMapper';
+import { useDynamicStyles } from 'hooks';
+import { Crumb } from './Crumb';
+
+export const Breadcrumbs = (props: BreadCrumbProps) => {
+  const { pathname } = props;
```

Comment:
pathname should probably just be defined here instead of passed as props.

Reply:
I passed as props to decouple the component from any dependencies on specific package libraries